pkg/operator/ceph/cluster/osd/deviceSet.go

```diff
@@ -76,7 +82,9 @@ func (c *Cluster) prepareStorageClassDeviceSets(config *provisionConfig) []rookv
 					ClaimName: pvc.GetName(),
 					ReadOnly:  false,
 				}
-				logger.Infof("successfully provisioned pvc %q for VolumeClaimTemplates %q for storageClassDeviceSet %q of set %v", pvc.GetName(), pvcTemplate.GetName(), storageClassDeviceSet.Name, i)
+				if newPVC {
+					logger.Infof("successfully provisioned pvc %q for VolumeClaimTemplates %q for storageClassDeviceSet %q of set %v", pvc.GetName(), pvcTemplate.GetName(), storageClassDeviceSet.Name, i)
```

Comment:
What about logging this inside `createStorageClassDeviceSetPVC` so we could avoid changing the signature completely? (not return the boolean)